### DISCUSSION TITLE
Change directory for saving Bookmarks, Notes, Tracks & Routes

### DISCUSSION THREAD

**Discussion by trymeouteh** [organicmaps/organicmaps#8634] [Ideas]
Please have a setting to allow users to change the directory on where there bookmarks, and (notes, tracks and routes once there a feature) on their device.

I would like to store my bookmarks in /Documents/Maps/Bookmarks/ personally.

This feature can allow users to setup apps like Syncthing or other cloud storage apps to sync and backup their bookmarks automatically

**Comment by pastk** [comment]
- https://github.com/organicmaps/organicmaps/pull/1871
